src/routes/home/contact-form.tsx

```diff
@@ -0,0 +1,205 @@
+import { useForm } from "react-hook-form";
+import { Button, Form } from "../../components";
+
+const emailMessageURL =
+  import.meta.env.MODE === "production"
+    ? "https://synthesis-workshop-backend-97f537f332bd.herokuapp.com/api/email-message"
+    : "http://localhost:8080/api/email-message";
+
+const nameRules = {
```

Comment:
I pulled all of this from form.stories.tsx, but removed the `import` statement. My reasoning is that if someone changes something in the storybook file in the future, it's a reasonable assumption that a change to that file shouldn't affect anything in the actual website since storybook is just a documentation / testing tool, so we shouldn't be importing things directly from storybook for use in the application